compiler/src/dotty/tools/dotc/typer/Implicits.scala

```diff
@@ -338,7 +337,7 @@ object Implicits:
       if monitored then record(s"check uncached eligible refs in irefCtx", refs.length)
       val ownEligible = filterMatching(tp)
       if isOuterMost then ownEligible
-      else combineEligibles(ownEligible, outerImplicits.uncachedEligible(tp))
+      else combineEligibles(ownEligible, outerImplicits.nn.uncachedEligible(tp))
```

Comment:
I used `uncheckedNN` in `def level` because I know `outerImplicits` is non-nullable immediately from the if condition. I should change `uncheckedNN` to `nn` in other places.